src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -147,7 +147,7 @@ RCLConsensus::Adaptor::relay(RCLCxPeerPos const& peerPos)
 }
 
 void
-RCLConsensus::Adaptor::relay(RCLCxTx const& tx)
+RCLConsensus::Adaptor::share(RCLCxTx const& tx)
 {
     // If we didn't relay this transaction recently, relay it to all peers
```

Comment:
I went with `share` in the generic context and left `relay` in the ripple specific implementation (`RCL*`). The thought is that the generic version can use a variety of routing strategies to send messages in support of the consensus algorithm, so `relay` was too specific.  However, since `rippled`'s overlay and hashrouter use `relay`, I stuck with that name. 

I agree that having two names is not ideal, but I felt the generic vs concrete boundary was an acceptable spot to diverge.